system_time_function.sql

```diff
@@ -1,27 +1,17 @@
 -- version 0.6.1
 
-CREATE OR REPLACE FUNCTION set_system_time(user_timestamp text)
-RETURNS text AS $$
+CREATE OR REPLACE FUNCTION set_system_time(user_timestamp timestamptz)
```

Comment:
@simoneb,
This function is now on par with the one in the original extension. Test case from the original extension is now passing for this.
The only thing missing for backward compatibility is, input with `YYYY-MM-DD HH24:MI:SS.MS.US` text format is no longer supported. All other text formats are supported.
I tried to make it work but Postgres throws an error when a text with above format is supplied to a parameter of `timestamptz` type.

Reply:
Yes correct. But there is a caveat to it. Let me put it this way.
The updated `set_system_time` function primarily works with a `timestamptz` input. It also has limited support for text input.

#### Text input still supported after the upgrade
- 2001-01-01 22:59:59
- 2001-02-01

#### Text input no longer supported after the upgrade
- 2001-02-01 22:59:59.001.000234

Hope this sounds good.

Reply:
ok that's perfect then. we can still mark this as a breaking change, but in practice the impact of the change is much smaller than it would be if the string format stopped being supported altogether.

can you please see if there is any documentation to update (in the readme basically)?

Reply:
Sure, looking into them now.